setup/SetupScript.sh

```diff
@@ -0,0 +1,26 @@
+#
+#  Original solution via StackOverflow:
+#    http://stackoverflow.com/questions/35802939/install-only-available-packages-using-conda-install-yes-file-requirements-t
+#
+
+#
+#  Install via `conda` directly.
+#  This will fail to install all
+#  dependencies. If one fails,
+#  all dependencies will fail to install.
+#
+
+echo Input desired virtual environment name
+read envName
+export PATH=~/anaconda3/bin:$PATH
+conda create -n $envName -y python=3.6 jupyter
+source activate $envName
```

Comment:
Please change to 'activate' for Windows users, and split into 2 installation scripts: one for OSX/ Linux, and one for Windows, thx man